packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -21,6 +21,22 @@ const ctx = jestContext.new().add(jestConsoleContext()).add(jestProcessContext()
 // To avoid the loading spinner locally
 process.env.CI = 'true'
 
+// describeIf is making eslint not happy about the names
+/* eslint-disable jest/no-identical-title */
+
+function sanitizeSQLServerIdName(schema: string) {
+  const schemaRows = schema.split('\n')
+  const regexp = new RegExp(/map: "PK__(.*)__(.*)"/)
+  const schemaRowsSanitized = schemaRows.map((row) => {
+    const match = regexp.exec(row)
+    if (match) {
+      row = row.replace(match[2], 'RANDOM_ID_SANITIZED')
+    }
+    return row
+  })
+  return schemaRowsSanitized.join('\n')
+}
```

Comment:
I believe `match[2]` is the table's field name that appears in the primary key constraint name, but an explanatory comment would be useful here.

Reply:
added